driver-core/src/main/java/com/datastax/driver/core/DateWithoutTime.java

```diff
@@ -0,0 +1,138 @@
+/*
+ *      Copyright (C) 2012-2015 DataStax Inc.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+package com.datastax.driver.core;
+
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.TimeZone;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Represents a Cassandra {@code DATE} data type.
+ * The {@code DATE} data type is available since Cassandra 2.2 via native protocol V4.
+ *
+ * @since 2.2
+ */
+public abstract class DateWithoutTime {
```

Comment:
We'll probably want as well:
* a constructor / factory method taking `year, month, day`
* and possibly one taking `yyyy-MM-dd` (although that's more likely to be used in tests)